src/domain/Car.js

```diff
@@ -0,0 +1,27 @@
+import { CAR } from "../constants/error.js";
+import {
+  CAR_NAME_MAX_LENGTH,
+  DRIVE_MIN_POSITION,
+} from "../constants/number.js";
+
+export default class Car {
+  name;
+  position;
+
+  constructor(name) {
+    this.checkName(name);
+    this.name = name;
+    this.position = 0;
+  }
```

Comment:
1. position 값을 default로 주는 것은 어떨까요?!
2. checkName 메서드 이후 name을 할당하는데, 컨텍스트를 긴밀하게 만들면 좋을 것 같습니다. checkName을 isValidName 등의 명칭으로 변경하고 boolean을 반환하게 하면 어떨까요?

 ```suggestion
 export default class Car {
  name;
  position = 0;

  constructor(name) {
    if(!this.invalidName(name)){
      throw new Error(CAR.NAME_LENGTH_EXCEED);
    }
    
    this.name = name;
  }
```